website/src/css/customTheme.css

```diff
@@ -62,7 +62,6 @@ div.inner {
 .image {
   padding: 75px 10px 75px 75px;
   height: 200px;
-  width: 200px;
```

Comment:
Hi @leslie-tsang I noticed that only remove `width` is not enough, you can try to remove `height`, just let the image adapt to the current container